### ISSUE TITLE
RequestError: unable to get local issuer certificate

### ISSUE THREAD

**Issue by amisskii** [containers/podman-desktop#9586]
### Bug description

I am encountering multiple issues related to extensions and authenticated container registry.

- Extension catalog not available:
   -  The extension catalog does not appear on the Extensions page.
   ![Image](https://github.com/user-attachments/assets/f7969804-9bfa-42f4-bf13-ed26aeac9738)

- Unable to download custom extensions:
  - Attempting to download custom extensions results in failure.
  ![Image](https://github.com/user-attachments/assets/fbb0a7fe-6a2f-426c-ba91-207da077e370)

- Cannot add a new private container registry:
  - Adding an authenticated container registry fails with the same error message.
  ![Image](https://github.com/user-attachments/assets/2f2d7224-09cb-47d2-8332-05bc250e3939)

The following error message appears across all the above actions: `Error: RequestError: unable to get local issuer certificate`

UPD: Everything works when I use the application with the tarball installation method.

### Operating system

Fedora 40

### Installation Method

Flatpak

### Version

next (development version)

**Comment by benoitf** [comment]
After further investigation it might be related to https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1781

(dependencies used when building the application in flatpak) so it's not related to a change in Podman Desktop codebase

**Comment by AlexonOliveiraRH** [comment]
I'm facing the same issue issue after updating it to the latest version with Flatpak:

![Image](https://github.com/user-attachments/assets/ad54b304-c4db-4012-85aa-2dc706370719)
![Image](https://github.com/user-attachments/assets/6b50c9f5-6836-49f1-8d31-fa08531f0687)
![Image](https://github.com/user-attachments/assets/ad29cdd4-9dca-498d-9fee-4f5c80bbb105)

But the tarball (`podman-desktop-1.13.2.tar.gz`) works perfectly.

**Comment by AlexonOliveiraRH** [comment]
@benoitf the issue you pointed out in GitLab is exactly the problem here. Check the workaround in my last comment there:

https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1781#note_2179404803

@amisskii test that workaround, please, and let us know the outcome.

**Comment by mecrobio** [comment]
Tried the workaround in Fedora 40 without success

**Comment by AlexonOliveiraRH** [comment]
> Tried the workaround in Fedora 40 without success

How did you do it?

**Comment by AlexonOliveiraRH** [comment]
Making it clear:

```
$ cd /var/lib/flatpak/runtime/org.freedesktop.Platform/x86_64/23.08/6c32cd7614d71747de14d2b2f384f19dc37017aea4ef8e3a5e75bf1c2e3bdc52/files/etc/pki/ca-trust/extracted/pem/directory-hash
$ sudo ln -s /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem ca-certificates.crt
$ sudo ln -s /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem ca-bundle.crt
```

**Comment by Erick555** [comment]
Note if you use user flatpak installation then the commands are:

```
$ cd ~/.local/share/flatpak/runtime/org.freedesktop.Platform/x86_64/23.08/6c32cd7614d71747de14d2b2f384f19dc37017aea4ef8e3a5e75bf1c2e3bdc52/files/etc/pki/ca-trust/extracted/pem/directory-hash
$ ln -s /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem ca-certificates.crt
$ ln -s /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem ca-bundle.crt

```

**Comment by eranco74** [comment]
> [@benoitf](https://github.com/benoitf) the issue you pointed out in GitLab is exactly the problem here. Check the workaround in my last comment there:
> 
> https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1781#note_2179404803
> 
> [@amisskii](https://github.com/amisskii) test that workaround, please, and let us know the outcome.

This workaround fixed it for me

**Comment by benoitf** [comment]
fixed in 1.13.3 for flathub
